sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -1151,7 +1147,7 @@ def visit_table(self, node):
         # detected, apply the default table width observed when using the v2
         # editor.
         if self.v2 and 'colwidths-given' in table_classes:
-            attribs['data-table-width'] = CONFLUENCE_DEFAULT_V2_TABLE_WIDTH
+            attribs['data-table-width'] = self._v2_table_width
```

Comment:
Note that this will only override/set the user-defined (or plugin default) only when column widths are defined on a table. If looking to be able to override defaults for all tables even non-width-hinted column tables, this would need adjustments.

Reply:
I do not believe it will cause issues under v2. We can always run the validation checks to see if it causes issues.

Maybe it would be good to introduce a `confluence_table_width` directive that operates in a similar fashion as the `highlight` directive does ([for example](https://github.com/sphinx-contrib/confluencebuilder/blob/61287ff7eaf0d7a79b3d677b2a532be9ce96a080/tests/unit-tests/datasets/code-block/code-block-highlight.rst?plain=1#L37)). So if a user wanted to override a single table's width on a document, they can do:

```
.. confluence_table_width: 500px

========= =========
Header 1  Header 2
========= =========
Row 1 Col 1 Row 1 Col 2
Row 2 Col 1 Row 2 Col 2
========= =========
```

Just food-for-thought. Does not need to be part of this pull request.